bor-mainnet.toml

```diff
@@ -551,3 +551,29 @@
 'v1-045000-045500-borspans.seg' = 'ae40aa774a8ce225d4b3b7a1b77592fc4373048c'
 'v1-045000-045500-headers.seg' = '5360bd1f51c275cb9844a16d70813a1440400d72'
 'v1-045000-045500-transactions.seg' = '669ebe60283b63e663e24e0aa7bc3e29c1d3d756'
+'v1-045500-046000-bodies.seg' = 'cca9947dbff5e228e92a748c27a3b7786548b71c'
+'v1-045500-046000-headers.seg' = '3e7f558763cec757db16df4caadb65e831b70508'
+'v1-045500-046000-transactions.seg' = 'dc7a59fc1737796d77197159ff7133e5e29413f9'
+'v1-046000-046100-borevents.seg' = '05dd13e7bda9a9b66dbd9b7e7c00fff104f8f2b4'
+'v1-046000-046100-borspans.seg' = 'c74a2354ca2a83c7c356772e8841c42bf6a11518'
+'v1-046000-046500-bodies.seg' = '19e795b0df22454f043453eac14576542b8e7c9e'
+'v1-046000-046500-headers.seg' = 'ce2468447939adc976fae5d4b792e2f9b226fae9'
+'v1-046000-046500-transactions.seg' = '4d660773516c994606126cc47e0792d2ad450dbb'
+'v1-046100-046200-borevents.seg' = '4d902fcad5a828418dd6628291dacedc449770cf'
```

Comment:
this is different range (100k) from the line above, is this expected?

Reply:
I don’t know. Have running node on it - no results yet.

Reply:
fixed by merging all files to 500K